package.json

```diff
@@ -34,6 +34,7 @@
     "eslint-config-airbnb": "^6.2.0",
     "eslint-plugin-react": "^4.2.3",
     "express": "^4.13.4",
+    "jquery": "^2.2.3",
```

Comment:
Because I included `clever-components`, we do. I could do it with raw buttons instead.